scripts/analysis.py

```diff
@@ -2,6 +2,7 @@
 import sys
 from pyne import nucname
 import numpy as np
+import numpy.ma as ma
```

Comment:
You don't seem to ever call for a masked array with ma. You don't need to import it if you never use it.